facades.md

```diff
@@ -0,0 +1,58 @@
+# Adding a new facade to third-party-web
+
+Lighthouse 7.0 includes a new audit to flag third-party 
```

Comment:
this paragraph feels like it might belong in lighthouse or web.dev pointing to this doc?

here it feels like just a sentence saying this powers lighthouse would work

Reply:
sure. i tweaked it.

since we're spread across LH report, web.dev docs, this doc, etc.. I still want a little bit of context everywhere. it's redundant but whatever.